scripts/find-maintainer

```diff
@@ -1,89 +0,0 @@
-#!/usr/bin/env python3
-# -*- coding: utf-8 -*-
-
```

Comment:
Did you delete this file on purpose?

Reply:
@avikivity Yes, once `MAINTAINERS` is removed, this script no longer makes sense.

Reply:
Can't the script just use the new file? Not everybody uses github, although I'm not sure anybody ever used this script at all.

Reply:
@denesb I did, once. Just to realize it was not up-to-date though :)

Reply:
@denesb There's a cost in porting `find-maintainers` to use the new format, and I am less than convinced it's worth it. For starters, I am not aware of anyone ever actually using the script. Second, on "not everybody uses Github" -- that ship has sailed a long time ago (maybe we left a few people stranded on the island of emails, but...). Third, there are existing tools out there that parse `CODEOWNERS`. See for example https://github.com/nmusolino/codeowners. Anyone that needs to parse this information outside GitHub can use one of them.

Reply:
@penberg I'm not insisting we do this porting, I was just curious why you decided it "no longer makes sense". I agree we shouldn't waste time on keeping this script alive at all cost.

Reply:
Merge ping!